sound/soc/sof/topology.c

```diff
@@ -1443,6 +1444,7 @@ static int sof_connect_dai_widget(struct snd_soc_component *scomp,
 				 */
 				if (!cpu_dai->playback_widget) {
 					cpu_dai->playback_widget = w;
+					dai->cpu_dai_name = cpu_dai->name;
```

Comment:
do we need a Fixes tag as well here?

Reply:
@plbossart I want to squash the first 2 commits and remove cpu_dai_name from struct snd_sof_dai altogether. With the new logic to use the playback/capture_widget assocuated with the CPU DAI, we dont really need this at all. What do you think?  But then its not really a fx anymore but rather a change in logic.

Reply:
Sounds good to me to remove something we don't need :-)